src/me/ryanhamshire/griefprevention/GriefPrevention.java

```diff
@@ -412,7 +414,14 @@ public static Player checkPlayer(CommandSource source) throws CommandException {
                                 .permission(GPPermissions.COMMAND_CLAIMFLAG)
                                 .arguments(GenericArguments.firstParsing(GenericArguments.flags().flag("-r", "r")
                                         .buildWith(GenericArguments.seq(optional(onlyOne(string(Text.of("flag")))),
-                                                optional(onlyOne(GenericArguments.remainingJoinedStrings(Text.of("value"))))))))
+                                                optional(GenericArguments.firstParsing(onlyOne(GenericArguments.choices(Text.of("value"), ImmutableMap.<String, Tristate>builder()
+                                                        .put("-1", Tristate.FALSE)
+                                                        .put("0", Tristate.UNDEFINED)
+                                                        .put("1", Tristate.TRUE)
+                                                        .put("false", Tristate.FALSE)
+                                                        .put("undefined", Tristate.UNDEFINED)
```

Comment:
To make it more user friendly I'd suggest using "default" or "inherit" rather then undefined.